docs/src/tut/mkt/index.md

```diff
@@ -0,0 +1,1323 @@
+
+* Line 60 publicizes the `decision` from line 58.
+
+* Line 61 concludes the `Buyer`'s steps with a `commit();`
+
+Finally, the result of the program is shown with the following code:
+
+```reach
+load: /examples/mkt-2-basic/index.rsh
+range: 63 - 63
+```
+
+* Line 63 executes the `showResult` `{!rsh} function` on both `{!rsh} Participants`.
+
+In the terminal, execute `{!cmd} ./reach run`.
+An output similar to what's printed below should display.
+
+```
+Welcome to the Market
+Contract info: {"type":"BigNumber","hex":"0x6d"}
+List of products for sale:
+1. Potatoes at 10 per bag.
+2. Carrots at 10 per bunch.
+3. Corn at 5 per ear.
+Buyer wants Potatoes
+Buyer agrees to buy 9 bags of Potatoes
+Seller agrees to sell 9 bags of Potatoes
+
+
+
+Welcome to the Market
+Contract info: {"type":"BigNumber","hex":"0x89"}
+List of products for sale:
+1. Potatoes at 10 per bag.
+2. Carrots at 10 per bunch.
+3. Corn at 5 per ear.
+Buyer wants Carrots
+Buyer agrees to buy 10 bunches of Carrots
+Seller agrees to sell 10 bunches of Carrots
+
+
+
+Welcome to the Market
+Contract info: {"type":"BigNumber","hex":"0x90"}
+List of products for sale:
+1. Potatoes at 10 per bag.
+2. Carrots at 10 per bunch.
+3. Corn at 5 per ear.
+Buyer wants Carrots
+Buyer agrees to buy 81 bunches of Carrots
+Seller agrees to sell 81 bunches of Carrots
+```
+
+:::note 
+`{!cmd} ./reach run` was executed 3 times to the 3 different outputs that you see above.
+
+You get one result each time that you execute `{!cmd} ./reach run`.
+:::
+
+This is a beautiful start. 
+You just created a simple DApp that makes a simple transaction happen.
+
+### {#tut-mkt-basic-Review} Review
+This section taught the basics of the Reach language and demonstrated how the final DApp to be built will work. You were able to learn that:
+
+* A `{!rsh} Participant` is a user.
+
+* An integer is represented by `{!rsh} UInt`. 
+
+* A string is represented by `{!rsh} Bytes(x)`.
+The integer (x) in braces defines how long the string may be.
+
+* A `{!rsh} function` in Reach is defined as `{!rsh} Fun([input], output)`.
+
+* `{!rsh} parseCurrency` `{!rsh} function` was provided by Reach to help convert regular currencies into currencies acceptable on a DApp.
+
+* Calling `{!rsh} interact` on something in Reach is the same thing as calling upon a `{!rsh} function` to execute the code within it.
+
+* A local step is an action that happens on a `{!rsh} Participant`'s machine. This step is used to execute codes that are only accessible to a particular `{!rsh} Participant`.
+
+* `{!rsh} declassify()` is used to make any secret `{!rsh} publish`able or shareable with other `{!rsh} Participant`s.
+
+* A contract is the same as a written agreement that must be signed by all `{!rsh} Participant`s before any transaction can take place. The `{!rsh} Participant` that initiates the transaction deploys or creates the contract while the other `{!rsh} Participant`(s) will be attached to it.
```

Comment:
transaction deploys or creates the contract <-- only need one: creates or deploys

Also, sentence needs proper punctuation.